guide/results-viewer-react/src/components/YamlUrls/index.tsx

```diff
@@ -308,3 +271,126 @@ export function Urls ({ data: { headers, ...data }, ...props }: UrlProps) {
 }
 
 export default Urls;
+
+interface RequestDetailsTabsProps extends HeadersViewProps, ResponseViewProps {}
+
+function RequestDetailsTabs ({ id, headersList, removeHeader, changeHeader, addHeader, responseCode, response }: RequestDetailsTabsProps): JSX.Element {
```

Comment:
When creating sub elements, it's a bit cleaner to put them in their own file and create a separate storybook for them.